src/components/Tooltip/components/TooltipContent/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { ReactElement, RefObject } from 'react';
+import ReactDOM from 'react-dom';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import clsx from 'clsx';
+import { Props } from './types';
+
+
+function TooltipContent({
+    className,
+    boundingRect,
+    anchorHeight,
+    anchorWidth,
+    anchor = 'top',
+    color = 'primary',
+    isVisible,
+    children,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const tooltipRef: RefObject<HTMLDivElement> = React.useRef(null);
+    const [tooltipPosition, setTooltipPosition] = React.useState();
+    const OVERFLOW_PRIORITY = ['left', 'right', 'bottom', 'top'];
+    const { top: tooltipTop, left: tooltipLeft } = tooltipPosition || {};
+    const { top: nodeTop, left: nodeLeft, bottom: nodeBottom, right: nodeRight } = boundingRect || {};
+    const tooltipClasses = clsx(
+        'brew-Tooltip__content',
+        [`brew-Tooltip--color-${color}`],
+        [`brew-Tooltip--anchor-${anchor}`],
+    );
+
+
+    // Set top and left values
+    React.useEffect(() => {
+        if (!tooltipRef.current || !boundingRect) return;
+        const tooltipEl = Array.from(tooltipRef?.current?.parentNode?.children ?? []);
+        const tooltipHeight: number = tooltipEl[0].clientHeight;
+        const tooltipWidth: number = tooltipEl[0].clientWidth;
```

Comment:
I'm a little concerned about this throwing an error when no children are present (since the fallback is a 0-length array, but then you pull the client height & width from the first element of said array).

Is there a use-case without children? If not, it might be better to just return if children are missing.